src/cc2olx/models.py

```diff
@@ -282,6 +286,16 @@ def get_resource_content(self, identifier):
                     )
                 )
                 return "html", {"html": html}
+            elif "web_resources" not in str(res_filename):
+                # Keep track of this file and later copy into static dir
+                self.extra_static_files.append(res_relative_path)
+                html = (
+                    '<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>'
+                    '</head><body><p><a href="{}" alt="{}">{}<a></p></body></html>'.format(
+                        "/static/" + res_relative_path, res_relative_path, res_relative_path
```

Comment:
Nit: `/static/` can be moved in a constant in this class and then use everywhere, else there can be a chance in future if we change the path we have to do multiple places

Reply:
Moved ``static`` to ``OLX_STATIC_DIR`` const.